lib/any_login/providers/devise.rb

```diff
@@ -3,6 +3,11 @@ module Provider
     module Devise
 
       module Controller
+        
+        extend ActiveSupport::Concern
```

Comment:
@saiqulhaq 
can you do similar for non-devise "providers", I think it's needed too.

Reply:
done https://github.com/igorkasyanchuk/any_login/pull/22/commits/6028bdcb7a6cdce9ca36302985bb94425dcd0e48